pkg/logs/parser/docker.go

```diff
@@ -3,22 +3,15 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2018 Datadog, Inc.
 
-package docker
+package parser
```

Comment:
I'd rather keep docker in its own package and implement the parser there than the reverse, because I feel like the pattern docker.DoXYZ(...) is better than parser.DoDockerXYZ(...), but that is debatable. We may need to review how we organize our packages so we don't end up with a mess, happy to chat about it.

Reply:
cc @ajacquemot for input

Reply:
looks nice when using the parser but I'd avoid having a docker package that only contains the parsing logic.